distribution/helm/deps.bzl

```diff
@@ -1,9 +1,8 @@
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 def deps():
-    http_archive(
+    git_repository(
```

Comment:
Let's tweak the ordering: declare the `name`, `remote` and then `commit`.

Reply:
done.

Reply:
Oh and also remove the `.git` suffix for the `remote` value. Our convention is to not add them.

You should look at other `deps.bzl` files in other packages to learn more about our coding convention.